bot/cogs/bot.py

```diff
@@ -15,6 +18,8 @@ class Bot:
 
     def __init__(self, bot: AutoShardedBot):
         self.bot = bot
+        self.allowed = {303906576991780866: 0, 303906556754395136: 0, 303906514266226689: 0, 267624335836053506: 0}
+        # stores allowed channels plus unix timestmp from last call
```

Comment:
comments do not go after the line they apply to. missing capital S. timestmp isn't a word.